docs/docs/building-a-blog.md

```diff
@@ -2,7 +2,63 @@
 title: Building a blog
 ---
 
-This is a stub. Help our community expand it.
 
-Please use the [Gatsby Style Guide](/docs/gatsby-style-guide/) to ensure your
-pull request gets accepted.
+### Building a blog
+
+Using gatsby-starter-blog you will build a full-featured blog. 
```

Comment:
I think it would make sense to include a link to the starter and the installation code so readers of the guide don't have to search for the starter!

Reply:
Link added